README.md

```diff
@@ -429,3 +429,80 @@ Similarly, you can run the integration test suite via
 ```bash
 $ python run_tests.py tests/integration/
 ```
+
+
+i18n compatibility
+==================
+
+According to [edX docs on XBlock i18n][edx-docs-i18n], LMS runtime is capable of supporting XBlock i18n and l10n. 
+To comply with l10n requirements, XBlock is supposed to provide translations in 
+`xblock_package_root/translations/locale_code/LC_MESSAGES/text.po` folder in GNU Gettext Portable Object file format.
+
+[edx-docs-i18n]: http://edx.readthedocs.io/projects/xblock-tutorial/en/latest/edx_platform/edx_lms.html#internationalization-support
+
+Drag and Drop v2 XBlock aims to comply with i18n requirements for Open edX platform, including a stricter set of 
+requirements for `edx.org` itself, thus providing the required files. So far only two translations are available:
+
+* Default English translation
+* Fake "Esperanto" translation used to test i18n/l10n.
+
+Updates to translated strings are supposed to be propagated to `text.po` files. Unfortunately, the process is mostly 
+manual. EdX [i18n_tools][edx-i18n-tools] might be helpful, but they are too tied to edx-platform, so as of Aug 2016 
+does not provide any benefits over manual approach.
+
+[edx-i18n-tools]: https://github.com/edx/i18n-tools
+
+In lieu of automated solution (i.e. edX i18n-tools supporting XBlock use case), one can use GNU Gettext toolkit
+to simplify manual updating `text.po` files.
+
+This command scrapes all the strings in all `*.py` files in `drag_and_drop_v2` folder and outputs `messages.po` file
+ in `drag_and_drop_v2` folder:
+
+```
+~/xblock-drag-and-drop-v2/drag_and_drop_v2$ find . -name="*.py" | xargs xgettext --language=python
```

Comment:
I think `-name="*.py"` should be `-name "*.py". I get an error with the `=` version.